packages/block-library/src/table/style.scss

```diff
@@ -31,12 +26,6 @@
 		// Table cannot be 100% width if it is aligned, so set
 		// width as auto.
 		width: auto;
-
-		td,
-		th {
-			// Aligned tables shouldn't scroll horizontally so we need their contents to wrap.
```

Comment:
While I did some alignment testing as can be seen in the gif, I'm a little curious about this comment. Do the contents need to wrap by break-all? Or can they just wrap naturally? I'm hoping my removal of this doesn't resurface an issue I'm unaware of. @jasmussen @talldan any thoughts?

Reply:
Changing this to overflow-wrap seems to be the better solution.

Reply:
I think there's an issue with `overflow-wrap`. If you type in some text with no spaces along and use a block alignment, the table goes off the screen:
<img width="675" alt="Screen Shot 2019-07-25 at 11 41 17 am" src="https://user-images.githubusercontent.com/677833/61844277-6d7fca00-aed1-11e9-8e9b-c243734c6487.png">

`word-break: break-word;` might be the best compromise. That still seems to wrap long words, but prefers breaking at word boundaries where possible.

Reply:
Thanks for that catch, @talldan! Using `word-break: break-all` was just so difficult for me b/c it broke a LOT of words, at least in Firefox. If it's not complicated, what if we used `overflow-wrap` for the default aligned Table block, and overwrite that with `break-word` for any Table block that uses alignments?

Reply:
👍 Yep, great idea.

Reply:
@mapk I think what you want to do here is update the current word-break: break-**all** to word-break: break-**word** instead of replacing with overflow-wrap. That should accomplish what you want without causing issues with long words not wrapping.

I see @talldan mentioned that above, but I know from working with this in the past it's easy to miss that distinction while reading things so thought I might point it out.